### ISSUE TITLE
octokit.issues.create problem

### ISSUE THREAD

**Issue by jaysee260** [octokit/octokit.js#823]
I am testing this on two different computers. __it works on my laptop but not on my desktop__.

I'm trying to create an issue on a repo, but I get an error when I run the code on my desktop. Since the same code works on my laptop (__the issue is actually created__), I have no idea about what's going on.

Here's the code

```js
const octokit =  require('@octokit/rest')();
const { token } = require('./config');

octokit.authenticate({
  type: 'oauth',
  token: token
});

octokit.issues.create({
  owner: 'org-name',
  repo: 'repo-name',
  title: 'first test',
  body: 'I am the first issue being created with the GitHub Node.js client!'
}).then(result => {
  console.log(result.data);
}).catch(error => {
  console.log(error);
})
```

and here's the error from the console

```shell
{ HttpError: {"message":"Not Found","documentation_url":"https://developer.github.com/v3/issues/#create-an-issue"}
    at response.text.then.message (/PROJECT_PATH/node_modules/@octokit/rest/lib/request/request.js:56:19)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
  name: 'HttpError',
  code: 404,
  status: 'Not Found',
  headers: 
   { 'access-control-allow-origin': '*',
     'access-control-expose-headers': 'ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval',
     connection: 'close',
     'content-encoding': 'gzip',
     'content-security-policy': 'default-src \'none\'',
     'content-type': 'application/json; charset=utf-8',
     date: 'Thu, 22 Mar 2018 21:54:52 GMT',
     'referrer-policy': 'origin-when-cross-origin, strict-origin-when-cross-origin',
     server: 'GitHub.com',
     status: '404 Not Found',
     'strict-transport-security': 'max-age=31536000; includeSubdomains; preload',
     'transfer-encoding': 'chunked',
     'x-accepted-oauth-scopes': '',
     'x-content-type-options': 'nosniff',
     'x-frame-options': 'deny',
     'x-github-media-type': 'github.squirrel-girl-preview',
     'x-github-request-id': 'EBCE:1C228:138DCEF:2CAA4CE:5AB4262C',
     'x-oauth-scopes': '',
     'x-ratelimit-limit': '5000',
     'x-ratelimit-remaining': '4983',
     'x-ratelimit-reset': '1521757305',
     'x-runtime-rack': '0.037243',
     'x-xss-protection': '1; mode=block' } }
```

I also tried, on my desktop, to use `octokit.repos.get` and it worked just fine. What is happening? Why is it that when I run `octokit.issues.create`, I get an Http error saying NOT FOUND?

A huge thank you in advance to anyone who can help !

**Comment by gr2m** [comment]
Usually a `404 NOT FOUND` means that you are not correctly authenticated.

Is it possible that the token is read out from an environment variable that is set on one computer but not the other?

If that’s not it, can you please run `node -v` and `npm ls @octokit/rest` on both and tell me if the outputs are the same?

**Comment by jaysee260** [comment]
output of `npm -v` on
- desktop: v.8.9.4
- laptop: v8.1.3

output of `npm ls @octokit/rest`
- desktop: `@octokit/rest@15.2.5`
- laptop: `@octokit/rest@14.0.9`

**Comment by gr2m** [comment]
is the repository you try to create the issue on private?

**Comment by jaysee260** [comment]
It's a public repository in an organization... though I just figured out the problem.

@gr2m: 

> Usually a 404 NOT FOUND means that you are not correctly authenticated.

I was using two different tokens, each with different scopes. The one on my laptop has writing privileges, whereas the one used on my desktop had no scopes. So I granted it the necessary ones and now it's working 😅 

Thank you for pointing me to the answer! 😄
